packages/grafana-runtime/src/components/AnnotationResultMapper.tsx

```diff
@@ -0,0 +1,201 @@
+import React, { PureComponent } from 'react';
```

Comment:
why is this in runtime package?

Reply:
I don't want to add it to `@grafana/ui` since it is a stop-gap until this can just be default behavior.  My goal is to get *something* so that I can easily use this same pattern/component for the immediate projects I am working on:
* influxdb
* adx
* github
* snowflake